### PR TITLE
Gh 17 create jobs page

### PR DESCRIPTION
**Pull request by joshmcintosh**
Please test **before** we break prod again thanks ;)

### CODE REVIEW

File: ccf/tests/test_models.py [hunk 1]
```diff
@@ -3,13 +3,25 @@
 from app.models import Job
 
 
+@pytest.fixture
```
Comment (Ross-Alexandra) [inline ccf/tests/test_models.py]: I would prefer this be in a test_utils.py file that way we can create users on command in any tests we may need to in the future.

Comment (joshmcintosh) [reply ccf/tests/test_models.py]: Addressed in latest commit.

File: ccf/tests/test_util.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+from django.contrib.auth.models import User
```
Comment (Ross-Alexandra) [inline ccf/tests/test_util.py]: Please call this file test_utils.py, theoretically there will be many utilities in this file, not just one.

Comment (joshmcintosh) [reply ccf/tests/test_util.py]: *sigh*

**Pull request merged**